assets/services.json

```diff
@@ -1298,6 +1298,15 @@
.129 1.772 1.051 1.05 1.678 2.401 1.764 3.804 0.087 1.415-0.384 2.712-1.324 3.653l-13.673 13.671 14.769 19.566a5.951 5.951 0 0 1 1.152 4.445 5.956 5.956 0 0 1-2.328 3.957 5.94 5.94 0 0 1-3.609 1.211 5.953 5.953 0 0 1-4.793-2.385l-14.071-18.644-2.082 2.082v13.091a6.01 6.01 0 0 1-6.002 6.003z\" fill=\"#fff\"/><path d=\"m67.765 145.72c-1.3036-0.80962-2.3328-1.9623-2.7608-3.0918-0.23216-0.61272-0.34917-7.051-0.34917-19.212v-18.29l-5.6724-0.10294c-5.5123-0.10004-5.7095-0.12753-6.9872-0.97399-3.349-2.2186-3.2812-7.1079 0.13096-9.4466l1.1859-0.81279 16.393-0.09958c11.065-0.06722 16.787 0.0154 17.605 0.25424 4.1759 1.2182 5.5604 6.1449 2.6129 9.2974-1.5183 1.6238-2.4937 1.842-8.2346 1.842h-5.2484l-0.0072 18.251c-0.0067 17.044-0.04716 18.337-0.61154 19.56-0.69918 1.5149-1.7749 2.5916-3.0854 3.0882-1.4498 0.54932-3.869 0.42122-4.971-0.26324z\" fill=\"#fff\" stroke-width=\".32737\"/><path d=\"m86.097 145.94c-2.6979-0.94565-3.6962-3.473-2.7532-6.9698 0.92264-3.421 15.209-40.389 16.064-41.568 1.8327-2.5274 5.0062-4.0205 7.69-3.618 1.6431 0.2464 4.1032 1.8182 5.3952 3.447 1.0582 1.3341 15.477 38.922 16.153 42.11 0.54775 2.5819 0.20846 4.5885-0.94338 5.5792-2.2205 1.91-7.0692 1.858-8.7451-0.0937-0.2919-0.33995-1.2494-2.4984-2.1279-4.7966l-1.5972-4.1785-18.539 0.173-1.6973 4.4194c-1.6714 4.352-1.7196 4.4307-3.1579 5.156-1.588 0.80078-4.0174 0.94493-5.7414 0.34065zm26.024-20.474c-3e-3 -0.33779-5.8718-16.868-6.0536-17.05-0.17968-0.17968-0.14321-0.27601-3.3981 8.9752-1.4624 4.1565-2.6589 7.7289-2.6589 7.9386 0 0.27639 1.666 0.38128 6.0563 0.38128 3.331 0 6.0554-0.11049 6.0543-0.24552z\" fill=\"#fff\" stroke-width=\".32737\"/><path d=\"m136.23 145.2c-0.69227-0.24682-1.632-0.79279-2.0882-1.2133-1.7568-1.6189-1.7273-1.2057-1.7273-24.221 0-21.087 8e-3 -21.367 0.69439-22.784 1.89-3.9042 7.3099-4.3837 9.9805-0.88294l0.94659 1.2408 0.0896 18.767 0.0896 18.767 6.785 2e-3c3.7318 1e-3 7.323 0.15124 7.9804 0.33386 1.5306 0.42514 3.4434 2.5205 3.739 4.0958 0.48283 2.5738-1.321 5.2736-4.0251 6.0245-1.9489 0.54121-20.889 0.43122-22.465-0.13047z\" fill=\"#fff\" stroke-width=\".32737\"/><path d=\"m168.92 145.79c-1.5165-0.8013-2.1683-1.4693-2.8216-2.8922-0.56394-1.2281-0.60176-2.651-0.6085-22.894-7e-3 -21.334 1e-3 -21.601 0.6872-23.019 1.7793-3.6756 6.5313-4.4019 9.2332-1.4113 0.6376 0.70574 1.3156 1.846 1.5066 2.534 0.19254 0.69339 0.34813 4.2405 0.34915 7.9595l2e-3 6.7087 8.757-8.7202c4.8164-4.7961 9.0788-8.8831 9.4721-9.0821 1.1168-0.5652 3.4238-0.43073 4.5667 0.26619 2.1911 1.336 3.4518 3.9907 2.9192 6.147-0.14753 0.59726-2.8065 3.5091-7.61 8.3337l-7.3833 7.4157 7.6133 10.08c4.1873 5.544 7.8148 10.562 8.0611 11.152 0.62487 1.4955 0.55879 2.9485-0.20596 4.5283-1.6356 3.3788-5.9468 4.3502-8.7686 1.9758-0.51739-0.43536-4.0965-4.9972-7.9535-10.137-3.857-5.1402-7.1694-9.3469-7.3608-9.3482-0.19137-1e-3 -0.73438 0.4028-1.2067 0.89794l-0.85873 0.90026-0.10277 7.5544-0.10277 7.5544-0.87866 1.3267c-1.5633 2.3604-5.0086 3.384-7.3055 2.1704z\" fill=\"#fff\" stroke-width=\".32737\"/></svg>"
```

Comment:
I like this icon more, it's smaller and simpler.

```
<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 24 24\" ><path d=\"M22.125 0H1.875C.839 0 0 .84 0 1.875v20.25C0 23.161.84 24 1.875 24h20.25C23.161 24 24 23.16 24 22.125V1.875C24 .839 23.16 0 22.125 0zM12 18.75c-.591 0-1.17-.041-1.732-.12-.562.396-3.813 2.679-4.12 2.722 0 0-.125.049-.232-.014s-.088-.229-.088-.229c.032-.22.843-3.018.992-3.533-2.745-1.36-4.57-3.769-4.57-6.513 0-4.246 4.365-7.688 9.75-7.688s9.75 3.442 9.75 7.688c0 4.245-4.365 7.687-9.75 7.687zM8.05 9.867h-.878v3.342c0 .296-.252.537-.563.537s-.562-.24-.562-.537V9.867h-.878a.552.552 0 0 1 0-1.101h2.88a.552.552 0 0 1 0 1.101zm10.987 2.957a.558.558 0 0 1 .109.417.559.559 0 0 1-.219.37.557.557 0 0 1-.338.114.558.558 0 0 1-.45-.224l-1.319-1.747-.195.195v1.227a.564.564 0 0 1-.562.563.563.563 0 0 1-.563-.563V9.328a.563.563 0 0 1 1.125 0v1.21l1.57-1.57a.437.437 0 0 1 .311-.126c.14 0 .282.061.388.167a.555.555 0 0 1 .165.356.438.438 0 0 1-.124.343l-1.282 1.281 1.385 1.835zm-8.35-3.502c-.095-.27-.383-.548-.75-.556-.366.008-.654.286-.749.555l-1.345 3.541c-.171.53-.022.728.133.8a.857.857 0 0 0 .357.077c.235 0 .414-.095.468-.248l.279-.73h1.715l.279.73c.054.153.233.248.468.248a.86.86 0 0 0 .357-.078c.155-.071.304-.268.133-.8l-1.345-3.54zm-1.311 2.443.562-1.596.561 1.596H9.376zm5.905 1.383a.528.528 0 0 1-.539.516h-1.804a.528.528 0 0 1-.54-.516v-3.82c0-.31.258-.562.575-.562s.574.252.574.562v3.305h1.195c.297 0 .54.231.54.515z\"/></svg>
```

Reply:
Icon has been updated :relaxed: